tests/framework/db/oci/type/BlobTest.php

```
@@ -98,46 +103,150 @@ public function testUpdateRoundTripsBinaryPayloadSizes(): void
+        }
+
+        self::assertSame(
+            $seed,
+            $this->readBlob(1),
+            'BLOB must stay untouched without a unique key.',
+        );
+    }
+
+    public function testThrowNotSupportedExceptionWhenBlobUpdateConditionIsNotHash(): void
+    {
+        $db = $this->getConnection();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)